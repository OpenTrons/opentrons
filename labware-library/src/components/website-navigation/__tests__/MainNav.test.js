// @flow
// tests for main navbar
import * as React from 'react'
import { shallow } from 'enzyme'

import { MainNav } from '..'

describe('MainNav', () => {
  test('component renders', () => {
    const tree = shallow(<MainNav />)

    expect(tree).toMatchSnapshot()
  })
})
