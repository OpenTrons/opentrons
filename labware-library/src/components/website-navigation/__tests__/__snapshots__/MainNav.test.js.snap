// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav component renders 1`] = `
<div
  className="main_nav_contents"
>
  <Logo />
  <NavList />
  <MenuButton />
</div>
`;
